numpy/core/_add_newdocs.py

```diff
@@ -5599,13 +5675,31 @@
 
     These are roughly ordered from least-to-most precision.
 
+    Examples
+    --------
+
+    >>> dt = np.dtype(str)
+    >>> dt.num
+    19
+
+    >>> dt = np.dtype(float)
+    >>> dt.num
+    12
+
     """))
 
 add_newdoc('numpy.core.multiarray', 'dtype', ('shape',
     """
     Shape tuple of the sub-array if this data type describes a sub-array,
     and ``()`` otherwise.
 
+    Examples
+    --------
+
+    >>> dt = np.dtype('4i4')
```

Comment:
I'd prefer to use the more verbose `('i4', 4)` syntax here

Reply:
Doing this results in shape `()` . Should I include that example?

Reply:
I think he meant `dt = np.dtype(('i4', 4))`